.github/workflows/deploy.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Build and deploy
+
+on:
+  push:
+    branches: ["dui3/alpha", "development", "main"] # Continuous delivery on every long-lived branch
```

Comment:
Currently "development" and "main" dedicated for DUI2, we should not trigger DUI3 installer on them